Nucleation_model_histogram.py

```diff
@@ -71,7 +74,14 @@ def residual(t, sol_vec):
     # Read out current state:
     A, C_Li, C_LiO2 = sol_vec[:3]
     n_p = sol_vec[3:]
-
+    N_tot = sum(n_p)
+    print(N_tot)
+    for key in pack_frac_dictionary:
+        if N_tot >= key:
+            pack_frac = pack_frac_dictionary[key]
+            print(pack_frac)
+            break
+    pack_frac = 0.875
```

Comment:
tbh I still don't understand what the `pack_frac_dictionary` represents, physically, nor what this loop is doing, specifically (does line 84 overwrite whatever the loop does?  Does the `break` statement exit the loop after the first time `N_tot >= key`? (I know I've gotten `break` wrong before, so maybe not).

Anyway, I'd rather merge this PR, and then we can discuss further.  I probably just need to go and read that paper.